### DISCUSSION TITLE
The Problem with "J" and "Y" letters in Uzbek Language.

### DISCUSSION THREAD

**Discussion by mmjnv** [espeak-ng/espeak-ng#1815] [General]
Hello everyone.

Our TTS Model is not pronouncing the words that contain these two letters: "J","Y".
What makes this problem complicated is that it is not happening with all words. For instance, the word "yaxshi" is sometimes pronounced properly, but sometimes pronounced as "jaxshi". It`s the same case with the words that contain the letter "J" which is pronounced like "Y". For example, the word "juda" is pronounced like "yuda".

If you have any suggestions or improvements, I would be happy to receive them. Thank you.

**Comment by jaacoppi** [comment]
Use the flag -X to check which rules are used for which words:

 espeak-ng -v uz -X "yaxshi"
 espeak-ng -v uz -X "juda"


You can change the rules in dictsource/uz_rules. Specifically, group y and group j.
